paymentapi/src/main/java/fi/hel/verkkokauppa/payment/model/Payment.java

```diff
@@ -179,4 +182,12 @@ public String getUserId() {
 	public void setUserId(String userId) {
 		this.userId = userId;
 	}
+
+	public String getPaytrailTransactionId() {
```

Comment:
Could you add lombok @getter and @setter to class? No need to add these later on